packages/tunnel/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import chalk from 'chalk';
+import express from 'express';
+import { createProxyMiddleware, responseInterceptor } from 'http-proxy-middleware';
+
+import Client from './client/index.js';
+import {
+  customUiSignInUrl,
+  logtoExperienceUrl,
+  logtoTunnelServiceUrl,
+  tunnelPort,
+} from './consts.js';
+
+const app = express();
+
+const client = new Client();
+await client.initSession();
```

Comment:
why do we need a client here? this should be implemented by developer

Reply:
The client is used to manually handle the redirect logic so that we can hijack the 302 redirect from response